rewrite-java/src/main/java/org/openrewrite/java/JavaVisitor.java

```diff
@@ -997,6 +997,24 @@ public J visitNewClass(J.NewClass newClass, P p) {
         return n;
     }
 
+    public J visitNullableType(J.NullableType nullableType, P p) {
```

Comment:
Info: any addition to a tree where the visitor is added should also have an associated visit in the ISO visitor.

Please add the visit to JavaIsoVisitor as well

Reply:
added to `JavaIsoVisitor`